source/guide_privatebin.rst

```diff
@@ -0,0 +1,204 @@
+.. highlight:: console
+
+.. author:: Nepomacs <https://github.com/Nepomacs/>
+
+.. categorize your guide! refer to the current list of tags: https://lab.uberspace.de/tags
+.. tag:: privacy
+.. tag:: lang-php
+.. tag:: web
+
+.. sidebar:: About
+
+  .. image:: _static/images/privatebin.png
+      :align: center
+
+##########
+PrivateBin
+##########
+
+.. tag_list::
+
+PrivateBin is a minimalist, open source online pastebin where the server has zero knowledge of pasted data.
+
+Data is encrypted and decrypted in the browser using 256bit AES in Galois Counter mode.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * PHP_
+  * :manual:`domains <web-domains>`
+
+License
+=======
+
+PrivateBin consists of PHP and JS code which was originally written by Sébastien Sauvage in 2012 and falls unter the Zlib/libpng license.
+All relevant legal information can be found in the Github_ repository of the project.
+
+
+Prerequisites
+=============
+
+We're using :manual:`PHP <lang-php>` in the stable version 7.4:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show node
+ Using 'PHP' version: '7.4'
+ [isabell@stardust ~]$
+
+
+
+The domain you want to use must be set up:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Download the source
+-------------------
+
+Check Github_ for the `latest release`_ and copy the download link to the .tar.gz file.
+Then ``cd`` to your :manual:`DocumentRoot <web-documentroot>` and use ``wget`` to download it. Replace the URL with the one you just copied.
+In the example the version is 1.2.4 which does not exist.
+
+.. code-block:: console
+ :emphasize-lines: 2
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/
+ [isabell@stardust isabell]$ wget https://github.com/PrivateBin/PrivateBin/archive/1.2.4.tar.gz
+ […]
+ Saving to: ‘1.2.4.tar.gz’
+
+ 100%[=================================================>] 3,172,029   3.45MB/s   in 0.9s
+
+ 2020-11-22 16:27:44 (8.32 MB/s) - ‘1.2.4.tar.gz’ saved [523648]
+ [isabell@stardust ~]$
+
+Untar the archive to the ``html`` folder and then delete it. Replace the version in the file name with the one you downloaded.
+
+.. code-block:: console
+ :emphasize-lines: 1,2
+
+ [isabell@stardust isabell]$ tar -xzf 1.2.4.tar.gz -C html/ --strip-components=1
+ [isabell@stardust isabell]$ rm 1.2.4.tar.gz
+ [isabell@stardust ~]$
+
+
+Activate the .htaccess file
+-----------------------------
+PrivateBin provides a .htaccess file, which blocks some known robots and link-scanning bots. Activate it by renaming it from ``.htaccess.disabled`` to ``.htaccess``.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html
```

Comment:
If you accept my previous suggestion, we're already in `html` and you can skip this line entirely.
```suggestion
```